### ISSUE TITLE
Open with Code not visible for unprivileged users

### ISSUE THREAD

**Issue by moander** [Microsoft/vscode#4439]
- VSCode Version: 0.10.11 
- OS Version: Windows Server 2008 R2

Steps to Reproduce:

1. Log in as a unprivileged user
2. Run the VS Code installer as Administrator and enter password
3. Tap both "Open with Code" checkboxes in the wizard
4. Right-click on a file or folder 

The "Open with Code" option is missing. I have not investigated this more than the reproduce steps above so it may be unrelated to the user being unprivileged.

**Comment by pfister** [comment]
It might be related to problem I'm seeing in linux x64: see #4434.  The 'Code' executable was renamed to 'code'.

**Comment by joaomoreno** [comment]
Duplicate of #98

**Comment by CalvinDale** [comment]
I experienced this issue on a Windows 10 machine. I installed VSC while logged-in to a user account (which prompted for Admin credentials). While all of the registry settings seem to have been added, the Windows Explorer context menu doesn't list the VSC entries. I hacked the registry on my Windows 7 machine at work using [these settings](https://gist.githubusercontent.com/CalvinDale/93ddc87c8f329d9048ff/raw/6a92519164fe3f0d0ddca43ebc7017952566ccc7/gistfile1.txt). And it worked, however that machine runs under an Admin account.
